Exercise_C.js

```diff
@@ -14,5 +14,12 @@ See README.md if you need help running your code.
 */
 
 for (let i = 1; i <= 30; i++) {
-  // CODE HERE
-}
+  if(i%3==0 && i%5==0){
+  return "Hong Kong";
+  } else if (i%3==0){
+  return "Hong";
+  } else if (i%5==0){
+    return "Kong"
+  }
+} 
+

```

Comment:
For the exercise, you have used return which does work when you want to return the result. But in this case, we want the answer to put output to the console.log, thus we should use console.log( ) and at the same time. You have not added the part where if the number does not divide by 3 or 5 or 3 and 5 then we want to return the number.
`   } else {
      console.log(i)
    }`